suuchi-rocksdb/src/test/scala/in/ashwanthkumar/suuchi/store/rocksdb/RocksDbStoreSpec.scala

```diff
@@ -2,29 +2,56 @@ package in.ashwanthkumar.suuchi.store.rocksdb
 
 import java.io.File
 
+import in.ashwanthkumar.suuchi.store.KV
 import org.apache.commons.io.FileUtils
-import org.scalatest.Matchers._
+import org.scalatest.Matchers.{be, convertToAnyShouldWrapper, have, startWith}
 import org.scalatest.{BeforeAndAfter, FlatSpec}
 
 class RocksDbStoreSpec extends FlatSpec with BeforeAndAfter {
-  val ROCKSDB_TEST_LOCATION = "/tmp/suuchi-rocks-test"
+
+  val ROCKSDB_TEST_LOCATION = "/tmp/suuchi-rocks-test/"
+
+  var db: RocksDbStore = _
+
   before {
     FileUtils.deleteDirectory(new File(ROCKSDB_TEST_LOCATION))
+    db = new RocksDbStore(RocksDbConfiguration.apply(ROCKSDB_TEST_LOCATION))
```

Comment:
@gsriram7 I thought we reverted this change - do we really need to extract this out as a `var db` and use it? Else was it breaking the tests?